camunda-formio-tasklist-vue/src/services/CamundaRest.ts

```diff
@@ -209,7 +209,7 @@ export const CamundaRest = {
 
```

Comment:
).then(async () => {
      await this.getGroupDetails();
      await this.reloadCurrentTask();
    });
    this.setGroup = null;
  }

Can this async and await be removed?

Reply:
addGroup () {
    CamundaRest.createTaskGroupByID(
      this.token,
      this.task.id!,
      this.bpmApiUrl,
      {
        userId: null,
        groupId: this.setGroup,
        type: "candidate",
      }
    ).then(() => {
      this.getGroupDetails();
      this.reloadCurrentTask();
    });
    this.setGroup = null;
  }

Reply:
removed

Reply:
@shuhaib-aot thanks for resolving. I am pretty sure more such issue are there in codebase. When you are working on Code optimization ticket, you can work on it.